gateway/api/views.py

```diff
@@ -858,6 +858,20 @@ def get_retrieve_queryset(self, pk):
 
         return Program.objects.filter(id=pk, instances=public_group).first()
 
+    def get_by_title_queryset(self, title, provider_name):
+        """
+        QuerySet to retrieve a specifc public programs in the catalog
+        """
+        public_group = Group.objects.filter(name=self.PUBLIC_GROUP_NAME).first()
+
+        if public_group is None:
+            logger.error("Public group [%s] does not exist.", self.PUBLIC_GROUP_NAME)
+            return []
```

Comment:
Probably a copy paste error but I understand that in this case it should return a None instead of an array, shouldn't it?

Reply:
These are exact same lines as https://github.com/Qiskit/qiskit-serverless/blob/e24c2a2543bf12b01588f1633b2f549a4285de50/gateway/api/views.py#L857
Either `[]` or `None` works in both methods.  I guess that they must be same.

Reply:
Yeah, I think this was a mistake from my side:
- `get_queryset` is fine
- `get_retrieve_queryset` I think it should return `None`

So `get_by_title_queryset` should return `None` too. Probably I copy-paste from `get_queryset` and I forgot to change it, yep!